webhook.yaml

```diff
@@ -5,10 +5,12 @@ clusters:
 - name: keycloak-server
   cluster:
     certificate-authority: /etc/origin/master/ca.crt
-    server: KEYCLOAK_URL/realms/master/protocol/openshift/token/token-review
+    server: KEYCLOAK_URL/realms/master/protocol/openid-connect/ext/openshift-token-review?client_id=token-review
 users:
 - name: api-server
   user:
+  client-certificate: /etc/origin/master/ca.crt
+  client-key: /etc/origin/master/ca.key
```

Comment:
this is broken as well

Reply:
I guess I would need a better explanation here. Let me tell you more about the idea I had for this...

So this part is about the API Server calling Keycloak. Since the API Server has both `ca.crt` and `ca.key` files at hand, and Keycloak uses CA Bundle for certificate verification, using `ca.key` and `ca.crt` is probably the easiest option for establishing Mutual TLS. 

Could you please tell me why this option is wrong? Also, how can I create a key and a certificate and make it available for the API Server (so that it is mounted probably somewhere in `/etc/origin/master/`).

Reply:
As for now, I will keep using CA (even though it's wrong). The Mutual TLS blocks @pedroigor's work. Once we agree on the best approach, I'll just switch to other key/cert pair.